README.md

```diff
@@ -1 +1,102 @@
+## Using Git in VS Code
+In this guide I will tell you how you, yes you could use git through VS code making your life significantly easier than before. Am I saying you will never have to use console technically as far as VS code is concerned yes with pushing and pulling. You do not have to use console except when you would need to roll back after pushing changes, let’s get started.
 
+### Step 1
+Open VS Code
```

Comment:
Can combine with the next step. Also, each sentence needs to end with proper punctuation.